src/Definitions/DefinitionGenerator.php

```diff
@@ -136,25 +133,18 @@ private function getDefinitionName(): string
     private function getModelFake(): ?Model
     {
         try {
-            DB::beginTransaction();
-
-            return factory(get_class($this->model))->create();
+            return factory(get_class($this->model))->make();
```

Comment:
Make is much safer, only creates an instance without making a connection to the db

Reply:
I  don't know if  it's more safer. Please, think about cases like that:

```php
$factory->define(App\Post::class, function ($faker) {
    return [
        'title' => $faker->title,
        'content' => $faker->paragraph,
        'user_id' => function () {
            return factory(App\User::class)->create()->id;
        }
    ];
});
```
If think we should to use transactions to avoid surprises...

Reply:
I checked here other reasons to use `create` instead of `make`. We done it to generate the `example` data to timestamps and id fields too. I don't know if it sufficient... bus was one of the reason.

Reply:
The test that check the `example` response is checking just if the result is not null. But I think that the correct behavior should check if is not empty. If apply this change the build will fail with the `make` method, because empty values in `created_at`,   `updated_at` na `id` properties.

Reply:
Ahh good point on the `user_id` example, I guess we can't avoid a transaction here. I'll change this back to what it was. Reasons I was trying to avoid create even with the transaction  are 1. to avoid having to make a connection to the db to generate static data, 2. it doesn't necessarily leave the db in an unchanged state (one example being the AUTO_INCREMENT value that remains unchanged even with a rollback) so it might just be some extra side effects that the user might not be expecting when running this. We might also want to add a config to disable this part by default and let the user manually turn it on if he chooses to, along with some prominent warnings not to run this in a prod environment (which should be obvious but we can't always guarantee that).

Reply:
I allow. I think we can maybe too ask user a different DB connection just to generate the docs... what about it?